adot/collector/config.yaml

```diff
@@ -17,7 +18,7 @@ service:
       exporters: [awsxray]
     metrics:
       receivers: [otlp]
-      exporters: [logging]
+      exporters: [logging, awsemf]
```

Comment:
Probably we can even remove the `logging` exporter, as the metric statistics message doesn't really serve any purpose if the metric itself is delivered. And if it can not be delivered, the error message will show up in the function's log group.

Reply:
Ageed